sql/api/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -139,7 +139,7 @@ abstract class DataFrameWriter[T] {
    *
    * @since 1.4.0
    */
-  def options(options: util.Map[String, String]): this.type = {
+  def options(options: Map[String, String]): this.type = {
```

Comment:
```suggestion
  def options(options: JMap[String, String]): this.type = {
```

Reply:
oh oops

Reply:
Since `Map` by default represents `scala.collection.immutable.Map` in Scala 2.13, I prefer the fix approach suggested by @yaooqinn  here.